game/domain/view/sectorview.lua

```diff
@@ -7,10 +7,17 @@ local TILE_H = 80
 local HALF_W = 10
 local HALF_H = 6
 
+-- FIXME: Tiles are not gotten from DB right now
+-- I just wanted to add code to draw from tiles instead of using rectangles
+-- I also tweaked the colors, but it's all provisory still
+local TEXTURE = love.graphics.newImage("assets/imgs/tiles.png")
+TEXTURE:setFilter("nearest", "nearest")
 local TILE_COLORS = {
-  [SCHEMATICS.FLOOR] = COLORS.FLOOR1,
-  [SCHEMATICS.EXIT] = COLORS.EXIT,
+  [SCHEMATICS.FLOOR] = love.graphics.newQuad(0, 0, TILE_W, TILE_H, TEXTURE:getDimensions()),
+  [SCHEMATICS.EXIT] = love.graphics.newQuad(80, 0, TILE_W, TILE_H, TEXTURE:getDimensions()),
+  shade = love.graphics.newQuad(160, 0, TILE_W, TILE_H, TEXTURE:getDimensions()),
 }
+local TILES = love.graphics.newSpriteBatch(TEXTURE, 512, "stream")
```

Comment:
I think my only complain is this part where you call functions from `love.graphics` "globally". With or without using the DB, it's generally better if this kind of code is in some sort of init function (in this case, the `SectorView`'s constructor should do).

Reply:
reasonable enough, on it